### DISCUSSION TITLE
Pressing the D key in the title screen should lead to daily challenge.

### DISCUSSION THREAD

**Discussion by IceBlossom** [ppy/osu#29069] [Ideas]
Title.
Right now it leads to... beatmap listing? of all things? Now that daily challenge is here, I feel like it would be a better fit. 
The C key isn't being used as well, so that could be used if it's a no-no to get rid of D leading to the beatmap listing.

**Comment by peppy** [comment]
please use <kbd>P</kbd> -> <kbd>D</kbd>

The <kbd>D</kbd> key at the main menu is a legacy binding for osu!direct which likely won't change.

**Reply by IceBlossom** [reply]
What about using the C key? That key doesn't have any functionality on the title screen so I feel like it could be used there.
